content/chainguard/chainguard-enforce/iam-groups/assumable-ids.md

```diff
@@ -0,0 +1,152 @@
+  value = chainguard_identity.<id-ref>.id
+}
+```
+
+The last two blocks you should include in a Terraform configuration are what apply a role-binding to the new identity. First, you need to include a `data` section to look up the role. In this example it looks up Chainguard's built-in `viewer` role.
+
+```
+data "chainguard_roles" "viewer" {
+  name = "viewer"
+}
+```
+
+Then you need to include another `resource` block to create the role-binding using the role that was just looked up. The identity will have the permissions of that role over the group specified within this block.
+
+```
+resource "chainguard_role-binding" "view-stuff" {
+  identity = chainguard_identity.<id-ref>.id
+  group    = chainguard_group.user-group.id
+  role     = data.chainguard_roles.viewer.items[0].id
+}
+```
+
+This means that the identity this Terraform configuration will create will only be able to view the resources tied to the same group the identity is tied to.
+
+Applying this configuration will create the assumable identity. You can follow our tutorial on [GitHub Actions example](/chainguard/chainguard-enforce/iam-groups/enforce-github-identity/) to create an assumable identity that can be used by a GitHub Actions workflow to interact with Chainguard Enforce. The Terraform file used in the linked tutorial is based closely on the template outlined here.
+
+
+## Managing identities with `chainctl`
+
+You can also set up an assumed identity using the `chainctl` command-line tool. Specifically, you can run the `chainctl iam identities create` subcommand, which uses the following syntax:
+
+```sh
+chainctl iam identities create <identity-name> \
+ --identity-issuer=<issuer of the identity> \
+--issuer-keys=<keys for the issuer> \
+--subject=<subject of the identity> \
+--group=<group name> \
+--role=<role>
+```
+
+As with Terraform, you must provide `chainctl` with certain information about the identity you want to create, including the issuer and subject of the identity, the role-bindings associated with the identity (if any), and the group under which the identity should be created.
+
+You can change an existing identity with the `update` command. The following example would update the identity's issuer.
+
+```sh
+chainctl iam identities update <identity-name> --identity-issuer=https://new-issuer.mycompany.com
+```
+
+To delete an identity, use the `delete` subcommand.
+
+```sh
+chainctl iam identities delete <identity-name>
+```
+
+For more detailed information on managing identities with `chainctl`, we encourage you to check out the [`chainctl` reference documentation](/chainguard/chainctl/chainctl-docs/chainctl_iam_identities/).
+
+
+## Assuming an identity
+
+Whether you create an identity with `chainctl` or with Terraform, Chainguard Enforce will generate a unique identification number tied to the identity. You can retrieve a list of all the Enforce identities you've created — along with the identification numbers — with the following command.
```

Comment:
Do we call these IDs numbers elsewhere? It sounds sort of strange to me.

Maybe "unique identifier" or "identifier"?

Reply:
Maybe this helps? https://edu.chainguard.dev/chainguard/chainguard-enforce/reference/events/#uidp-identifiers - it is only on the events page, but maybe it could be split out into a standalone reference page?

Reply:
UIDP identifiers sounds right, thanks Jamon

Reply:
Thanks all, I was struggling finding this word. "The identity's identification number" sounded real bad to me, but UIDP is way better